addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1880,28 +1878,27 @@ options.registry.collapse = options.Class.extend({
     _createIDs: function () {
         var time = new Date().getTime();
         var $tab = this.$target.find('[data-toggle="collapse"]');
-
         // link to the parent group
         var $tablist = this.$target.closest('.accordion');
-        var tablist_id = $tablist.attr('id');
-        if (!tablist_id) {
-            tablist_id = 'myCollapse' + time;
-            $tablist.attr('id', tablist_id);
+        var tablistId = $tablist.attr('id');
+        if (!tablistId) {
+            tablistId = 'myCollapse' + time;
+            $tablist.attr('id', tablistId);
         }
-        $tab.attr('data-parent', '#' + tablist_id);
-        $tab.data('parent', '#' + tablist_id);
+        $tab.attr('data-parent', '#' + tablistId);
+        $tab.data('parent', '#' + tablistId);
```

Comment:
I don't understand the purpose of this id and data-parent attributes ? For what are they used ? Can we get rid of it ?

Reply:
data-parent allow you to have a real accordeon...

When you open the second, the first (all other than current click) is automatically closed.
So yes it is still required imo.

Else if you drop 2 accordeon, you will have 2x the same id which on is not really html compliant.
And when you open the second tab of second accordeon, it will probably close all tab of the first one.

Reply:
> When you open the second, the first (all other than current click) is automatically closed.
So yes it is still required imo.

When I test on a runbot this is not the behavior we have. You can open all the tabs you want. There is no automotic closing of the other tabs. May be something to fix here @qsm-odoo or is it the intended behavior?

> Else if you drop 2 accordeon, you will have 2x the same id which on is not really html compliant.

Yes! Fortunately it doesn't seem to be a problem here. But probably because the automatic closing doesn't work...

Reply:
in this case it is the data-parent and id the problem probably:

Hsre an example of expected result:

https://7583251-master-all.runbot35.odoo.com/page2#
-> as you can see, you have 2 distinct accordeon, where each one can only have 1 open

And the bugged version if you keep the same id on both:
https://7583251-master-all.runbot35.odoo.com/
-> As you can see, you can open several tab on the second but not the first

>  it doesn't seem to be a problem here. 

Yes it is a problem, we should continue to avoid this when it is easy, so just keep the id rename ;)
Imagine you want to make a link to this accordion. The anchor will not work since it is duplicated

Reply:
> May be something to fix here @qsm-odoo or is it the intended behavior?

Of course something to fix :grimacing: